source/_patterns/01-atoms/icons/icon.twig

```diff
@@ -1,9 +1,17 @@
+{% if icon.width and icon.height %}
+  {% set height_width = 'width-' ~ icon.width ~ ' height-' ~ icon.height %}
```

Comment:
Let's remove the "width-" and "height-" prefixes, so templates can pass custom height/width, maxw, minh.

Same for the "text-" prefix below. Templates may want to pass a custom color, ie. jcc-text-primary.

Reply:
Sounds good. We figured that the customization would be done at the SCSS level since we want to discourage variance but good with this approach even if we prefer no variance from PL.

Will adjust my component once Nelson is done.

Reply:
I've gone ahead and simplified and taken out conditionals.